doc/admin/deployment.md

```diff
@@ -1,1115 +1,848 @@
-Starting SPDK v18.07-pre / DPDK 18.02.0 initialization...
-[ DPDK EAL parameters: spdk -c 0x1 --file-prefix=spdk234203216 --base-virtaddr=0x200000000000 --proc-type=auto ]
-EAL: Detected 72 lcore(s)
-EAL: Auto-detected process type: PRIMARY
-2019/03/28 12:28:07 main.go:79: debug: Switching control log level to DEBUG
-boro-45.boro.hpdd.intel.com 2019/03/28 12:28:07 config.go:121: debug: Active config saved to /home/tanabarr/projects/daos_m/utils/config/examples/.daos_server.active.yml (read-only)
-EAL: Detected 72 lcore(s)
-Starting SPDK v18.07-pre / DPDK 18.02.0 initialization...
-[ DPDK EAL parameters: spdk -c 0x1 --file-prefix=spdk290246766 --base-virtaddr=0x200000000000 --proc-type=auto ]
-EAL: Auto-detected process type: PRIMARY
-EAL: No free hugepages reported in hugepages-1048576kB
-EAL: No free hugepages reported in hugepages-1048576kB
-EAL: Multi-process socket /home/tanabarr/.spdk234203216_unix
-EAL: Probing VFIO support...
-EAL: Multi-process socket /home/tanabarr/.spdk290246766_unix
-EAL: Probing VFIO support...
-EAL: PCI device 0000:81:00.0 on NUMA socket 1
-EAL:   probe driver: 8086:2701 spdk_nvme
-EAL: PCI device 0000:81:00.0 on NUMA socket 1
-EAL:   probe driver: 8086:2701 spdk_nvme
-Starting SPDK v18.07-pre / DPDK 18.02.0 initialization...
-[ DPDK EAL parameters: daos -c 0x1 --file-prefix=spdk290246766 --base-virtaddr=0x200000000000 --proc-type=auto ]
-EAL: Detected 72 lcore(s)
-Starting SPDK v18.07-pre / DPDK 18.02.0 initialization...
-[ DPDK EAL parameters: daos -c 0x1 --file-prefix=spdk234203216 --base-virtaddr=0x200000000000 --proc-type=auto ]
-EAL: Auto-detected process type: SECONDARY
-EAL: Detected 72 lcore(s)
-EAL: Auto-detected process type: SECONDARY
-EAL: Multi-process socket /home/tanabarr/.spdk234203216_unix_141938_a591f56066dd7
-EAL: Probing VFIO support...
-EAL: WARNING: Address Space Layout Randomization (ASLR) is enabled in the kernel.
-EAL:    This may cause issues with mapping memory into secondary processes
-EAL: Multi-process socket /home/tanabarr/.spdk290246766_unix_23680_14e5a164a3bd1db
-EAL: Probing VFIO support...
-EAL: WARNING: Address Space Layout Randomization (ASLR) is enabled in the kernel.
-EAL:    This may cause issues with mapping memory into secondary processes
-EAL: PCI device 0000:81:00.0 on NUMA socket 1
-EAL:   probe driver: 8086:2701 spdk_nvme
-boro-44.boro.hpdd.intel.com 2019/03/28 12:28:11 main.go:188: debug: DAOS server listening on 0.0.0.0:10001
-DAOS I/O server (v0.0.2) process 141938 started on rank 1 (out of 2) with 1 target xstream set(s).
-EAL: PCI device 0000:81:00.0 on NUMA socket 1
-EAL:   probe driver: 8086:2701 spdk_nvme
-boro-45.boro.hpdd.intel.com 2019/03/28 12:28:11 main.go:188: debug: DAOS server listening on 0.0.0.0:10001
-DAOS I/O server (v0.0.2) process 23680 started on rank 0 (out of 2) with 1 target xstream set(s).
-```
-
-</p>
-</details>
-
-
-### Basic Workflow

```

Comment:
Unfortunately, this patch is reverting changes made by Tom.

Reply:
thanks for catching this